2023-05-17 12:45:26,737 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 12:45:26,739 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 12:45:26,739 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 12:45:26,750 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 12:45:26,751 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 12:45:26,751 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 12:45:26,757 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 12:45:26,759 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 12:45:27,761 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 13:22:25,497 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 13:22:25,498 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 13:22:25,498 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 13:22:25,502 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 13:22:25,503 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 13:22:25,503 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 13:22:25,508 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 13:22:25,509 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 13:22:26,510 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 14:18:16,893 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 14:18:16,897 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 14:18:16,898 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 14:18:16,917 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 14:18:16,919 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 14:18:16,919 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 14:18:16,944 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 14:18:16,954 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 14:18:17,966 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 14:22:01,307 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 14:22:01,311 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 14:22:01,312 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 14:22:01,332 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 14:22:01,334 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 14:22:01,334 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 14:22:01,362 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 14:22:01,373 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 14:22:02,381 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 14:27:13,970 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 14:27:13,973 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 14:27:13,974 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 14:27:13,990 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 14:27:13,992 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 14:27:13,992 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 14:27:14,014 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 14:27:14,022 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 14:27:15,030 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 14:56:31,869 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 14:56:31,872 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 14:56:31,872 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 14:56:31,890 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 14:56:31,891 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 14:56:31,892 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 14:56:31,916 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 14:56:31,925 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 14:56:32,932 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 15:34:45,717 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 15:34:45,719 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 15:34:45,720 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 15:34:45,738 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 15:34:45,740 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 15:34:45,740 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 15:34:45,766 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 15:34:45,776 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 15:34:46,785 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 15:36:08,765 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 15:36:08,768 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 15:36:08,769 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 15:36:08,788 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 15:36:08,790 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 15:36:08,790 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 15:36:08,813 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 15:36:08,823 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 15:36:09,839 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 15:37:47,696 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 15:37:47,699 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 15:37:47,699 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 15:37:47,715 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 15:37:47,716 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 15:37:47,717 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 15:37:47,738 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 15:37:47,748 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 15:37:48,764 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 15:52:41,509 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 15:52:41,512 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 15:52:41,513 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 15:52:41,530 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 15:52:41,531 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 15:52:41,532 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 15:52:41,554 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 15:52:41,564 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 15:52:42,580 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 15:59:39,961 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 15:59:39,964 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 15:59:39,964 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 15:59:39,980 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 15:59:39,981 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 15:59:39,981 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 15:59:40,004 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 15:59:40,012 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 15:59:41,025 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 21:14:13,403 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 21:14:13,404 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 21:14:13,405 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 21:14:13,418 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 21:14:13,418 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 21:14:13,418 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 21:14:13,429 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 21:14:13,433 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 21:14:14,448 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 22:07:39,062 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 22:07:39,064 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 22:07:39,064 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 22:07:39,073 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 22:07:39,074 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 22:07:39,074 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 22:07:39,087 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 22:07:39,092 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 22:07:40,095 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 22:17:25,145 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 22:17:25,147 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 22:17:25,147 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 22:17:25,158 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 22:17:25,159 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 22:17:25,160 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 22:17:25,174 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 22:17:25,178 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 22:17:26,187 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 22:19:39,195 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 22:19:39,197 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 22:19:39,197 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 22:19:39,205 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 22:19:39,206 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 22:19:39,206 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 22:19:39,217 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 22:19:39,223 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 22:19:40,237 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 22:27:47,650 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 22:27:47,651 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 22:27:47,651 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 22:27:47,661 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 22:27:47,662 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 22:27:47,662 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 22:27:47,673 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 22:27:47,677 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 22:27:48,693 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 22:29:31,368 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 22:29:31,370 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 22:29:31,371 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 22:29:31,380 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 22:29:31,380 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 22:29:31,380 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 22:29:31,396 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 22:29:31,401 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 22:29:32,416 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 22:30:15,260 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 22:30:15,262 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 22:30:15,262 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 22:30:15,270 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 22:30:15,271 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 22:30:15,271 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 22:30:15,282 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 22:30:15,287 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 22:30:16,295 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 22:37:11,848 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 22:37:11,850 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 22:37:11,850 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 22:37:11,860 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 22:37:11,861 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 22:37:11,861 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 22:37:11,872 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 22:37:11,876 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 22:37:12,882 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-17 22:38:57,515 - D:\project\python\EdgeFlow\core\context.py[line:87] - INFO: 初始化配置文件
2023-05-17 22:38:57,516 - D:\project\python\EdgeFlow\core\context.py[line:91] - INFO: 检查系统环境
2023-05-17 22:38:57,516 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活BaseContext服务---->
2023-05-17 22:38:57,526 - D:\project\python\EdgeFlow\core\context.py[line:106] - INFO: 正在激活Gui服务---->
2023-05-17 22:38:57,527 - D:\project\python\EdgeFlow\core\context.py[line:124] - INFO: 启动服务
2023-05-17 22:38:57,527 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动BaseContext服务
2023-05-17 22:38:57,538 - D:\project\python\EdgeFlow\core\context.py[line:126] - INFO: 启动Gui服务
2023-05-17 22:38:57,543 - D:\project\python\EdgeFlow\core\context.py[line:128] - INFO: 服务启动完成
2023-05-17 22:38:58,547 - D:\project\python\EdgeFlow\core\context.py[line:130] - INFO: 启动服务监听
2023-05-20 12:37:17,683 - D:\Python\EdgeFlow\core\context.py[line:107] - INFO: 初始化配置文件
2023-05-20 12:37:17,683 - D:\Python\EdgeFlow\core\context.py[line:111] - INFO: 检查系统环境
2023-05-20 12:37:17,684 - D:\Python\EdgeFlow\core\context.py[line:126] - INFO: 正在激活BaseContext服务---->
2023-05-20 12:37:17,695 - D:\Python\EdgeFlow\core\context.py[line:126] - INFO: 正在激活Gui服务---->
2023-05-20 12:37:17,695 - D:\Python\EdgeFlow\core\context.py[line:144] - INFO: 启动服务
2023-05-20 12:37:17,696 - D:\Python\EdgeFlow\core\context.py[line:146] - INFO: 启动BaseContext服务
2023-05-20 12:37:17,701 - D:\Python\EdgeFlow\core\context.py[line:146] - INFO: 启动Gui服务
2023-05-20 12:37:17,703 - D:\Python\EdgeFlow\core\context.py[line:148] - INFO: 服务启动完成
2023-05-20 12:37:18,710 - D:\Python\EdgeFlow\core\context.py[line:150] - INFO: 启动服务监听
2023-05-20 13:20:31,634 - D:\Python\EdgeFlow\core\context.py[line:107] - INFO: 初始化配置文件
2023-05-20 13:20:31,634 - D:\Python\EdgeFlow\core\context.py[line:111] - INFO: 检查系统环境
2023-05-20 13:20:31,636 - D:\Python\EdgeFlow\core\context.py[line:126] - INFO: 正在激活BaseContext服务---->
2023-05-20 13:20:31,639 - D:\Python\EdgeFlow\core\context.py[line:126] - INFO: 正在激活Gui服务---->
2023-05-20 13:20:31,640 - D:\Python\EdgeFlow\core\context.py[line:126] - INFO: 正在激活WebCam服务---->
2023-05-20 13:22:36,425 - D:\Python\EdgeFlow\core\context.py[line:107] - INFO: 初始化配置文件
2023-05-20 13:22:36,425 - D:\Python\EdgeFlow\core\context.py[line:111] - INFO: 检查系统环境
2023-05-20 13:22:36,426 - D:\Python\EdgeFlow\core\context.py[line:126] - INFO: 正在激活BaseContext服务---->
2023-05-20 13:22:36,430 - D:\Python\EdgeFlow\core\context.py[line:126] - INFO: 正在激活Gui服务---->
2023-05-20 13:22:36,430 - D:\Python\EdgeFlow\core\context.py[line:126] - INFO: 正在激活WebCam服务---->
2023-05-20 13:28:26,138 - D:\Python\EdgeFlow\core\context.py[line:109] - INFO: 初始化配置文件
2023-05-20 13:28:26,139 - D:\Python\EdgeFlow\core\context.py[line:113] - INFO: 检查系统环境
2023-05-20 13:28:26,139 - D:\Python\EdgeFlow\core\context.py[line:128] - INFO: 正在激活BaseContext服务---->
2023-05-20 13:28:26,143 - D:\Python\EdgeFlow\core\context.py[line:128] - INFO: 正在激活Gui服务---->
2023-05-20 13:28:26,144 - D:\Python\EdgeFlow\core\context.py[line:128] - INFO: 正在激活WebCam服务---->
2023-05-20 13:28:26,144 - D:\Python\EdgeFlow\core\context.py[line:146] - INFO: 启动服务
2023-05-20 13:28:26,145 - D:\Python\EdgeFlow\core\context.py[line:148] - INFO: 启动BaseContext服务
2023-05-20 13:28:26,151 - D:\Python\EdgeFlow\core\context.py[line:148] - INFO: 启动Gui服务
2023-05-20 13:28:26,153 - D:\Python\EdgeFlow\core\context.py[line:148] - INFO: 启动WebCam服务
2023-05-20 13:28:26,155 - D:\Python\EdgeFlow\core\context.py[line:150] - INFO: 服务启动完成
2023-05-20 13:28:27,169 - D:\Python\EdgeFlow\core\context.py[line:152] - INFO: 启动服务监听
2023-05-20 13:28:27,169 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:27,170 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:27,505 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:27,506 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:27,858 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:27,858 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:28,247 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:28,247 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:28,608 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:28,609 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:28,949 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:28,949 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:29,313 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:29,313 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:29,665 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:29,665 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:30,035 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:30,036 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:30,423 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:30,424 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:30,795 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:30,796 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:31,138 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:31,138 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:31,468 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:31,468 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:31,824 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:31,825 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:32,182 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:32,182 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:32,522 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:32,523 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:32,868 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:32,868 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:33,215 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:33,215 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:33,563 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:33,563 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:33,929 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:33,930 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:34,294 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:34,294 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:34,641 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:34,643 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:35,006 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:35,007 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:35,336 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:35,337 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:35,701 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:35,702 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:36,075 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:36,075 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:36,444 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:36,445 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:36,796 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:36,796 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:37,145 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:37,145 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:37,513 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:37,513 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:28:37,863 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:28:37,863 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:51,731 - D:\Python\EdgeFlow\core\context.py[line:109] - INFO: 初始化配置文件
2023-05-20 13:30:51,732 - D:\Python\EdgeFlow\core\context.py[line:113] - INFO: 检查系统环境
2023-05-20 13:30:51,732 - D:\Python\EdgeFlow\core\context.py[line:128] - INFO: 正在激活BaseContext服务---->
2023-05-20 13:30:51,736 - D:\Python\EdgeFlow\core\context.py[line:128] - INFO: 正在激活Gui服务---->
2023-05-20 13:30:51,737 - D:\Python\EdgeFlow\core\context.py[line:128] - INFO: 正在激活WebCam服务---->
2023-05-20 13:30:51,738 - D:\Python\EdgeFlow\core\context.py[line:146] - INFO: 启动服务
2023-05-20 13:30:51,738 - D:\Python\EdgeFlow\core\context.py[line:148] - INFO: 启动BaseContext服务
2023-05-20 13:30:51,743 - D:\Python\EdgeFlow\core\context.py[line:148] - INFO: 启动Gui服务
2023-05-20 13:30:51,746 - D:\Python\EdgeFlow\core\context.py[line:148] - INFO: 启动WebCam服务
2023-05-20 13:30:51,748 - D:\Python\EdgeFlow\core\context.py[line:150] - INFO: 服务启动完成
2023-05-20 13:30:52,759 - D:\Python\EdgeFlow\core\context.py[line:152] - INFO: 启动服务监听
2023-05-20 13:30:52,759 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:52,759 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:53,106 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:53,106 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:53,472 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:53,473 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:53,819 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:53,820 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:54,176 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:54,177 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:54,500 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:54,500 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:54,843 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:54,844 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:55,218 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:55,218 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:55,570 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:55,570 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:55,941 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:55,942 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:56,332 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:56,333 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:56,667 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:56,668 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:57,062 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:57,062 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:57,432 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:57,433 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:57,803 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:57,803 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:58,184 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:58,184 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:58,575 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:58,576 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:58,926 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:58,926 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:59,301 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:59,301 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:30:59,667 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:30:59,667 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:00,011 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:00,011 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:00,348 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:00,348 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:00,699 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:00,700 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:01,054 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:01,054 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:01,408 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:01,408 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:01,769 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:01,770 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:02,126 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:02,127 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:02,550 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:02,551 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:02,941 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:02,941 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:03,337 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:03,338 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:03,699 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:03,699 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:04,075 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:04,075 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:04,425 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:04,426 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:04,763 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:04,763 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:44,347 - D:\Python\EdgeFlow\core\context.py[line:109] - INFO: 初始化配置文件
2023-05-20 13:31:44,347 - D:\Python\EdgeFlow\core\context.py[line:113] - INFO: 检查系统环境
2023-05-20 13:31:44,348 - D:\Python\EdgeFlow\core\context.py[line:128] - INFO: 正在激活BaseContext服务---->
2023-05-20 13:31:44,352 - D:\Python\EdgeFlow\core\context.py[line:128] - INFO: 正在激活Gui服务---->
2023-05-20 13:31:44,352 - D:\Python\EdgeFlow\core\context.py[line:128] - INFO: 正在激活WebCam服务---->
2023-05-20 13:31:44,353 - D:\Python\EdgeFlow\core\context.py[line:146] - INFO: 启动服务
2023-05-20 13:31:44,353 - D:\Python\EdgeFlow\core\context.py[line:148] - INFO: 启动BaseContext服务
2023-05-20 13:31:44,358 - D:\Python\EdgeFlow\core\context.py[line:148] - INFO: 启动Gui服务
2023-05-20 13:31:44,360 - D:\Python\EdgeFlow\core\context.py[line:148] - INFO: 启动WebCam服务
2023-05-20 13:31:44,363 - D:\Python\EdgeFlow\core\context.py[line:150] - INFO: 服务启动完成
2023-05-20 13:31:45,374 - D:\Python\EdgeFlow\core\context.py[line:152] - INFO: 启动服务监听
2023-05-20 13:31:45,374 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:45,374 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:45,699 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:45,699 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:46,039 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:46,039 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:46,366 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:46,366 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:46,690 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:46,690 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:47,013 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:47,013 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:47,346 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:47,346 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:47,694 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:47,694 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:48,024 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:48,025 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:31:48,381 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:31:48,382 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:32:13,771 - D:\Python\EdgeFlow\core\context.py[line:109] - INFO: 初始化配置文件
2023-05-20 13:32:13,772 - D:\Python\EdgeFlow\core\context.py[line:113] - INFO: 检查系统环境
2023-05-20 13:32:13,772 - D:\Python\EdgeFlow\core\context.py[line:128] - INFO: 正在激活BaseContext服务---->
2023-05-20 13:32:13,777 - D:\Python\EdgeFlow\core\context.py[line:128] - INFO: 正在激活Gui服务---->
2023-05-20 13:32:13,777 - D:\Python\EdgeFlow\core\context.py[line:128] - INFO: 正在激活WebCam服务---->
2023-05-20 13:32:13,778 - D:\Python\EdgeFlow\core\context.py[line:146] - INFO: 启动服务
2023-05-20 13:32:13,778 - D:\Python\EdgeFlow\core\context.py[line:148] - INFO: 启动BaseContext服务
2023-05-20 13:32:13,783 - D:\Python\EdgeFlow\core\context.py[line:148] - INFO: 启动Gui服务
2023-05-20 13:32:13,785 - D:\Python\EdgeFlow\core\context.py[line:148] - INFO: 启动WebCam服务
2023-05-20 13:32:13,787 - D:\Python\EdgeFlow\core\context.py[line:150] - INFO: 服务启动完成
2023-05-20 13:32:14,799 - D:\Python\EdgeFlow\core\context.py[line:152] - INFO: 启动服务监听
2023-05-20 13:32:14,799 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:32:14,799 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:32:15,146 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:32:15,146 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:32:15,490 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:32:15,492 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:32:15,814 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:32:15,815 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:32:16,155 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:32:16,156 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:32:16,492 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:32:16,492 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:32:16,839 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:32:16,840 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
2023-05-20 13:32:17,215 - D:\Python\EdgeFlow\core\context.py[line:160] - ERROR: WebCam服务异常退出
2023-05-20 13:32:17,215 - D:\Python\EdgeFlow\core\context.py[line:165] - ERROR: WebCam服务异常退出，尝试重启
